### DISCUSSION TITLE
1.x ArrayTable 如何自定义render

### DISCUSSION THREAD

**Discussion by Zhou-Bill** [alibaba/formily#1649] [Q&A 1.x] [answered]
在ant-design 中 table 的columns 下写
```js
[{
    title: '姓名',
    dataIndex: 'name',
    render: (text, record) => text
 },
{
    title: '姓名/性别',
    dataIndex: 'sex',
    render: (text, record) => `${record.name}/${record.sex}`
 }]
```
如果拿record 可以拿到当前行的所有值，
在arrayTable 应该怎么做
[源码链接](https://github.com/alibaba/formily/blob/formily_v1/packages/antd-components/src/array-table/index.tsx)

源码这个地方，不清楚应该怎么做，是要写对应的schema?
![image](https://user-images.githubusercontent.com/13554001/123400734-a8686c00-d5d8-11eb-9fbf-c63546687261.png)

**Comment by Zhou-Bill** [comment]
只能够将数据 重新组合，然后渲染上去？

**Comment by janryWang** [comment, marked as answer]
不能自定义render，需要自己写自定义组件
